### DISCUSSION TITLE
Siteconfig is not defined on initial load.

### DISCUSSION THREAD

**Discussion by palicea** [requarks/wiki#3544] [Error / Bug Report]
### Actual behavior
Is not loading correctly everytime I open the browser. It only loads after a refresh is made to the site. The portal is published on IIS, and using the latest version of wiki.js. 

Siteconfig is not loaded when it tries to use it on the app.js file.
    n("./client/themes lazy recursive ^\\.\\/.*\\/scss\\/app\\.scss$")("./" + siteConfig.theme + "/scss/app.scss"),
![image](https://user-images.githubusercontent.com/13722675/92005687-c3988100-ed11-11ea-833e-d37a29859087.png)

### Expected behavior
The default website page should load. 

### Steps to reproduce the behavior
1. Access the website through the browser on the main url site.
2. unknown authentication  strategy jws error page.

**Comment by Smankusors** [comment]
please, where's the host information?

**Comment by palicea** [comment]
Azure VM
Windows Server 2012 R2
IIS 8.5
